Makefile

```diff
@@ -1,13 +1,9 @@
 # Makefile for pvc-autoresizer
 
-ENVTEST_K8S_VERSION = 1.19.2
-KUBEBUILDER_VERSION = 3.1.0
-CTRLTOOLS_VERSION = 0.6.0
-CTRLRUNTIME_VERSION = 0.8.3
-HELM_VERSION = 3.5.0
-HELM_DOCS_VERSION = 1.5.0
-
-export ENVTEST_K8S_VERSION
+ENVTEST_K8S_VERSION = 1.22.1
```

Comment:
Using Kubernetes 1.22.1 with envtest because envtest doesn't provide 1.22.4.

```
$ ./bin/setup-envtest list
(installed)  v1.22.1  linux/amd64
(available)  v1.22.1  linux/amd64
(available)  v1.22.0  linux/amd64
(available)  v1.21.4  linux/amd64
(available)  v1.21.2  linux/amd64
(available)  v1.20.2  linux/amd64
(available)  v1.19.2  linux/amd64
(available)  v1.19.0  linux/amd64
(available)  v1.17.9  linux/amd64
(available)  v1.16.4  linux/amd64
(available)  v1.15.5  linux/amd64
(available)  v1.14.1  linux/amd64
(available)  v1.13.1  linux/amd64
(available)  v1.12.3  linux/amd64
(available)  v1.11.0  linux/amd64
(available)  v1.10.1  linux/amd64
```